patterns/gitleaks/8.18.2/98-general.toml

```diff
@@ -326,7 +326,7 @@
       '''(?i)[\"\'][\s\w]+(?:password|secret|token)(?:_|-)?(?:access(?:_|-)?)?(?:key)?:\s*?[\"\']''',
       # Ignore Bitnami Sealed Secrets - This has the potential to cause false
       # negatives but out of all of our data so far I haven't seen one.
-      '''\bAg[A-Za-z0-9+\/]{256,}''',
+      '''[^A-Za-z0-9+\/]Ag[A-Za-z0-9+\/]{256,}''',
```

Comment:
Once you upgrade to support multiple allowlists, this can be a separate rule targeting the secret.

Reply:
Ack ack. :saluting_face: 

Would there be benefits other than scoping to specific file extensions? (Just confirming that's the main reason to break it out)

Reply:
> Would there be benefits other than scoping to specific file extensions? (Just confirming that's the main reason to break it out)

You _could_ constrain it to JSON and YAML; I was actually thinking it would simplify matching the pattern whole secret.

```regexp
^Ag[A-Za-z0-9+\/]{256,}$
```

Reply:
Ooo fair point. Yeah when we split out the allowlists into multiple and bump versions then I'll go in and refactor a lot of these. 

I hadn't considered the fact that multiple allowlists each get their own regex scope. That makes that feature even more handy for me. ^_^